### PR TITLE
Fix repo reference 

### PR DESCRIPTION
**Pull request by yaocw2020**
Issue: https://github.com/harvester/harvester/issues/727
Solution: 
- replace rancher reference with harvester
- update Kubernetes version to keep the same with harvester

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ steps:
       dockerfile: package/Dockerfile
       password:
         from_secret: docker_password
-      repo: "rancher/harvester-network-controller"
+      repo: "harvester/harvester-network-controller"
```
Comment (gitlawr) [inline .drone.yml:48]: Keep docker images unchanged

**Review by gitlawr — CHANGES_REQUESTED** [review]

File: pkg/apis/network.harvesterhci.io/v1beta1/zz_generated_register.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ limitations under the License.
 package v1beta1
 
 import (
-	network "github.com/rancher/harvester-network-controller/pkg/apis/network.harvesterhci.io"
+	network "github.com/harvester/harvester-network-controller/pkg/apis/network.harvesterhci.io"
```
Comment (gitlawr) [inline pkg/apis/network.harvesterhci.io/v1beta1/zz_generated_register.go:24]: move to the generated commit

File: scripts/package [hunk 3]
```diff
@@ -9,7 +9,7 @@ SUFFIX="-${ARCH}"
 cd $(dirname $0)/..
 
 TAG=${TAG:-${VERSION}${SUFFIX}}
-REPO=${REPO:-rancher}
+REPO=${REPO:-harvester}
```
Comment (gitlawr) [inline scripts/package:12]: Keep docker images unchanged

File: pkg/apis/network.harvesterhci.io/v1beta1/zz_generated_register.go (hunk 2 above)
Comment (yaocw2020) [reply pkg/apis/network.harvesterhci.io/v1beta1/zz_generated_register.go:24]: Sad, I always miss this file.

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**